images.CI/macos/select-datastore.ps1

```diff
@@ -54,14 +54,14 @@ function Select-DataStore {
 
     # 1. Name starts with ds-local-Datastore
     # 2. FreespaceGB > 400 Gb
-    # 3. VM count on a datastore < 2
+    # 3. Choose a datastore with the minimal VM count < 2
 
     Write-Host "Start Datastore selection process..."
     $allDatastores = Get-Datastore -Name $templateDatastore | Where-Object { $_.State -eq "Available" }
     $buildDatastore = $allDatastores | Where-Object { $_.FreeSpaceGB -ge $thresholdInGb } | Where-Object {
```

Comment:
Could you please split this very long pipe to a few variables?

Reply:
yep, done